packages/documentation/package.json

```diff
@@ -6,18 +6,16 @@
     "start": "astro dev",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro",
-    "docs:graphql:generate:backend": "./node_modules/.bin/graphql-markdown ../backend/src/graphql/schema.graphql > src/schema-dumps/backend_raw_schema.md",
-    "docs:graphql:generate:auth": "./node_modules/.bin/graphql-markdown ../auth/src/graphql/schema.graphql > src/schema-dumps/auth_raw_schema.md"
+    "astro": "astro"
   },
   "dependencies": {
     "@astrojs/react": "^2.3.2",
     "@astrojs/starlight": "^0.7.3",
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "astro": "2.10.14",
+    "astro-graphql-plugin": "^0.0.6",
     "graphql": "^16.8.0",
-    "graphql-markdown": "^7.0.0",
```

Comment:
If we are getting rid of `graphql-markdown` in the same PR we can remove the "Schema Types" files that were generated from the plugin